code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -270,6 +270,11 @@
 		return 0
 	else if(istype(card.loc,/mob))
 		var/mob/holder = card.loc
+		var/datum/belly/inside_belly //VOREStation edit.
+		inside_belly = check_belly(card) //VOREStation edit.
+		if(inside_belly) //VOREStation edit.
+			src << "There is no room to unfold in here. You're good and stuck." //VOREStation edit.

```

Comment:
to_chat

Y'know the more stuff you add, the more stuff I'mma ask you to change :p This is the whole point of doing PRs as a discrete single change/update and doing other stuff in others.